### ISSUE TITLE
NTR: Lactate: no rise with ischemic exercise test (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8239]
New term request Lactate: no rise with ischemic exercise test (f301): Lactic acid  UMLS: 1968732

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000526
- label: Failure to elevated lactate upon ischemic exercise test
- parents: [Abnormal glucose homeostasis (HP:0011014)](https://hpo.jax.org/app/browse/term/HP:0011014)
- pmid: PMID:3476851
- def: Failure to elevate lactate in blood washed out from ischemically exercised muscles as assayed by the forearm ischemic exercise test.
- comment: The forearm ischemic exercise test (FIET) has been accepted as an essential clinical laboratory test in the evaluation of patients suspected to have McArdle's disease and other disorders of muscle metabolism. Painful contracture during ischemic exercise and failure to elevate blood lactate duringthe FIET are regarded as the basic screening tests for disorders of anaerobic metabolism in muscle.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8239
- synonyms:
